content/blog/2022/cq-syntax.md

```diff
@@ -0,0 +1,365 @@
+---
+title: Use the Right Container Query Syntax
+sub: Size queries are stable, and shipping in browsers this summer
```

Comment:
@mirisuzanne Should we use a non relative time here since this article may be referenced for a while?